### PR TITLE
Add MKW-SP info to NW4R Exception Handler

### PR DESCRIPTION
**Pull request by riidefi**
Example output: `0.1.1 (Release) b7171ee (Built Feb 15 2022 at 18:58:02, 10.2.0)`

### CODE REVIEW

**Comment by riidefi** [comment]
Output
```
--------------------------------
MKW-SP v0.1.1 (Release) REV e8d4f66
Region: PAL, System: Dolphin 5.0-15993
Built Feb 15 2022 at 20:52:52, GCC 10.2.0
--------------------------------
******** EXCEPTION OCCURRED! ********
FrameMemory:90087EE0H
--------------------------------
CONTEXT:80347498H  (DSI EXCEPTION)
SRR0:   802245E8H   SRR1:0000B032H
DSISR:  42000000H   DAR: 00000096H```

File: payload/nw4r/db/dbException.c [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    case 'E':
+        region = "NTSC-U";
+        break;
+    case 'J':
+        region = "NTSC-J";
+        break;
+    case 'K':
+        region = "NTSC-K";
+        break;
+    }
```
Comment (stblr) [inline payload/nw4r/db/dbException.c:24]: We already have a `REGION` macro.

File: configure.py [hunk 2]
```diff
@@ -10,6 +10,15 @@
...
 
+import subprocess
+
+# https://stackoverflow.com/questions/14989858/get-the-current-git-hash-in-a-python-script/14989911#14989911
+def get_git_revision_hash() -> str:
+    return subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode('ascii').strip()
+
+def get_git_revision_short_hash() -> str:
+    return subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD']).decode('ascii').strip()
+
```
Comment (stblr) [inline configure.py:21]: I don't know if this is easily doable but could we detect if there are uncommitted changes in the working directory?

**Review by stblr — CHANGES_REQUESTED** [review]

File: configure.py (hunk 2 above)
Comment (riidefi) [reply configure.py]: Done, example output:
```--------------------------------
MKW-SP v0.1.1 (Release) REV f88bcd6
Region: PAL, System: Dolphin 5.0-15993*
Built Feb 16 2022 at 03:16:38, GCC 10.2.0
Changed files: 'configure.py, payload/game/host_system/RkSystem.S, payload/nw4r/db/dbException.c, payload/sp/Host.h, payload/sp/Version.h'
--------------------------------
******** EXCEPTION OCCURRED! ********
FrameMemory:900008C0H
```

**Pull request merged**